src/LEGO.AsyncAPI.Bindings/Kafka/TopicConfigurationObject.cs

```diff
@@ -27,12 +29,17 @@ public class TopicConfigurationObject : IAsyncApiElement
         /// <summary>
         /// The delete.retention.ms configuration option.
         /// </summary>
-        public int? DeleteRetentionMiliseconds { get; set; }
+        public int? DeleteRetentionMilliseconds { get; set; }
 
         /// <summary>
         /// The max.message.bytes configuration option.
         /// </summary>
         public int? MaxMessageBytes { get; set; }
+        
+        /// <summary>
+        /// The custom.configs properties configuration option.
```

Comment:
Probably worth adding into doc string the context that this is mostly to facilitate vendor specific configs AND to make it clear that there's no "clever" processing going on, I say this because some vendor specifics can have very weird formatting with potentially error prone characters that need to be escaped so it would be good for users to understand that they are responsible for sanitizing whatever goes into this dictionary.